test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerConfigFactoryTests.cs

```diff
@@ -273,29 +285,43 @@ public void GetWorkerProcessCount_Tests(bool defaultWorkerConfig, bool setProces
         [Fact]
         public void GetWorkerProcessCount_ThrowsException_Tests()
         {
-            JObject processCount = new JObject();
-            processCount["ProcessCount"] = -4;
-            processCount["MaxProcessCount"] = 10;
-            processCount["ProcessStartupInterval"] = "00:10:00";
-            processCount["SetProcessCountToNumberOfCpuCores"] = false;
-
-            JObject workerConfig = new JObject();
-            workerConfig[WorkerConstants.ProcessCount] = processCount;
+            JsonElement workerConfig = CreateWorkerConfig(-4, 10, "00:10:00", false);
 
             var config = new ConfigurationBuilder().Build();
             var testLogger = new TestLogger("test");
             RpcWorkerConfigFactory rpcWorkerConfigFactory = new RpcWorkerConfigFactory(config, testLogger, _testSysRuntimeInfo, _testEnvironment, new TestMetricsLogger(), _testWorkerProfileManager);
             var resultEx1 = Assert.Throws<ArgumentOutOfRangeException>(() => rpcWorkerConfigFactory.GetWorkerProcessCount(workerConfig));
             Assert.Contains("ProcessCount must be greater than 0", resultEx1.Message);
 
-            processCount["ProcessCount"] = 40;
+            workerConfig = CreateWorkerConfig(40, 10, "00:10:00", false);
             var resultEx2 = Assert.Throws<ArgumentException>(() => rpcWorkerConfigFactory.GetWorkerProcessCount(workerConfig));
             Assert.Contains("ProcessCount must not be greater than MaxProcessCount", resultEx2.Message);
 
-            processCount["ProcessStartupInterval"] = "-800";
-            processCount["ProcessCount"] = 10;
-            var resultEx3 = Assert.Throws<ArgumentOutOfRangeException>(() => rpcWorkerConfigFactory.GetWorkerProcessCount(workerConfig));
-            Assert.Contains("The TimeSpan must not be negative", resultEx3.Message);
+            workerConfig = CreateWorkerConfig(10, 10, "-800", false);
+            var resultEx3 = Assert.Throws<JsonException>(() => rpcWorkerConfigFactory.GetWorkerProcessCount(workerConfig));
+            Assert.Contains("value could not be converted to System.TimeSpan", resultEx3.Message);
+        }
+
+        private static JsonElement CreateWorkerConfig(int processCount, int maxProcessCount, string processStartupInterval, bool setProcessCountToCores)
```

Comment:
Do we have any tests for a bad or missing worker.config.json to test that we fail gracefully?

Reply:
The `GetWorkerProcessCount_ThrowsException_Tests` test validate incorrect JSON (only certain properties matches some expected value) is throwing an exception. We don't have any other tests validating the JSON. Missing worker config.json is a use case we validate for linux use cases. we strip out all language workers except the one needed for the image's runtime during the final image build phase.